HW_2/task_3.py

```diff
@@ -0,0 +1,19 @@
+"""
+Задайте список размерностью N. Каждый элемент списка вычисляется выражением (1 + 1 / n) ** n,
+где n – позиция (не индекс) элемента в списке, причем 1 < n < N.
+Выведите сумму элементов списка. Ответ округлите до сотых.
+Ввод: значение типа <int>/ Вывод: значение типа <float>
+Пример: 1/2.0  2/4.25  3/6.62
+"""
+
+n = int(input("Введите число n: "))
+my_list = []
+for i in range(1, n + 1):    # Position, not index
+    my_list.append((1 + 1 / i) ** i)
+print(round(sum(my_list), 2))
+
+exit()
```

Comment:
Не советуй подобным образом прерывать работу программы. Но если уж вам так сильно этого хочется, то лучше делай так
```suggestion
sys.exit()
```
С импортом соответствующей библиотеке, разумеется.